packages/fx-core/src/plugins/resource/frontend/configs.ts

```diff
@@ -109,6 +80,67 @@ export class FrontendConfig {
       });
   }
 
+  static getStorageName(ctx: PluginContext, getFromArmOutput = false): string {
+    let result = isArmSupportEnabled()
+      ? getStorageAccountNameFromResourceId(
```

Comment:
Since this method throw error, we need to handle it.

Reply:
The error will be thrown and handled by caller.

Reply:
Fixed.